src/video_core/renderer_vulkan/vk_graphics_pipeline.h

```diff
@@ -70,8 +94,8 @@ class VKGraphicsPipeline final {
 
     const VKDevice& device;
     VKScheduler& scheduler;
-    const FixedPipelineState fixed_state;
     const u64 hash;
+    GraphicsPipelineCacheKey cache_key;
```

Comment:
```suggestion
    const GraphicsPipelineCacheKey key;
    const u64 hash;
```
(You'll have to reorder the constructor.)